### PR TITLE
i18n: add Simplified Chinese translation

### PR DESCRIPTION
**Pull request by zombie110year**

### CODE REVIEW

**Comment by Herschel** [comment]
Hello, and welcome!

#1796 was opened slightly earlier. Could you compare your work and suggest any improvements there? Thank you!

**Comment by zombie110year** [comment]
I didn't check existing PR before working, but there were small difference, take what you prefer.

File: web/packages/extension/build/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "message": "查找当前标签页时发生错误。"
+    },
+    "status_message_init": {
+        "message": "在此标签页上检查 Ruffle 状态。"
+    },
+    "status_result_running": {
+        "message": "Ruffle 已加载并正在当前标签页播放 Flash 内容。"
+    },
+    "status_result_optout": {
+        "message": "Ruffle 未加载因为当前标签页标注自己不兼容。"
```
Comment (imba-tjd) [inline web/packages/extension/build/_locales/zh_CN/messages.json:27]: 原文是page不是tab，我觉得叫页面比较好。

File: web/packages/extension/build/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "status_result_disabled": {
+        "message": "Ruffle 未加载因为被用户禁用。"
+    },
+    "status_result_error": {
+        "message": "在查询 Ruffle 中此标签页实例时发生错误。"
```
Comment (imba-tjd) [inline web/packages/extension/build/_locales/zh_CN/messages.json:33]: @Herschel "An error occurred when querying the current tab's instance of Ruffle."
Does is mean "(current tab's instance) of Ruffle" or "current tab's (instance of Ruffle)"?

Comment (Herschel) [reply web/packages/extension/build/_locales/zh_CN/messages.json:33]: current tab's (instance of Ruffle)

File: web/packages/extension/build/_locales/zh_CN/messages.json (hunk 1 above)
Comment (zombie110year) [reply web/packages/extension/build/_locales/zh_CN/messages.json]: 已修改

File: web/packages/extension/build/_locales/zh_CN/messages.json (hunk 2 above)
Comment (zombie110year) [reply web/packages/extension/build/_locales/zh_CN/messages.json]: I have modified the commit.

```diff
- 在查询 Ruffle 中此标签页实例时发生错误。
+ 在查询此标签页中的 Ruffle 实例时发生错误。
```

**Comment by Herschel** [comment]
Thank you!

**Pull request merged**